client/src/components/HowToPlayModal.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { Modal } from "./index";
+import "./HowToPlayModal.css";
+
+function HowToPlayModal({ isActive, onClickClose }) {
+  return (
+    <>
```

Comment:
This react fragment `<>` does nothing here. It is only needed when a component's has more than one child element. 

ex. this will not work: 

```
<div></div>
<span></span>
```

this will work: 
```
<>
<div></div>
<span></span>
</>
```

but this also would work
```
<div>
  <div></div>
  <span></span>
</div>
```